src/Cache/QueryCacheProfile.php

```diff
@@ -117,9 +117,7 @@ public function getCacheKey()
      */
     public function generateCacheKeys($sql, $params, $types, array $connectionParams = [])
     {
-        if (isset($connectionParams['password'])) {
-            unset($connectionParams['password']);
-        }
+        unset($connectionParams['password'], $connectionParams['url']);
```

Comment:
Should this be in a separate commit? I don't see the link with sensitive parameters. Also, does this have security implications?

Reply:
At this point, the URL should be resolved to individual parameters. If it's still in there, it's 100% redundant. But you are right, I don't need this particular change. I'll move it into a separate PR.

Reply:
As you wish, but a separate commit is good enough.

Reply:
I've reverted the changes to this file.